bioblend/cloudman/__init__.py

```diff
@@ -724,7 +728,9 @@ def _make_get_request(self, url, parameters=None, timeout=None):
         if parameters is None:
             parameters = {}
         req_url = '/'.join((self.cloudman_url, 'root', url))
-        r = requests.get(req_url, params=parameters, auth=("", self.password), timeout=timeout)
+        # need to set the user "ubuntu" for auth, and indicate verify=False for HTTPS
+        # connections to self-signed certificates
+        r = requests.get(req_url, params=parameters, auth=("ubuntu", self.password), timeout=timeout, verify=False)
```

Comment:
Can you make `verify` also configurable (as `use_ssl`)?

Reply:
OK, will do. I'm a bit puzzled why I need to add the ubuntu user here - how did this ever work without it? Any clue?

Reply:
I notice @nsoranzo adding verify and timeout to get requests elsewhere using data-members... perhaps it would be more appropriate to add data members self.verify, self.authuser (naming?) to CloudManInstance and have these set in the constructor or elsewhere for later use? I presume the Cloudman auth user is a property of the base instance configured in whatever cloud and may need to be configurable (ec2-user, ubuntu, centos, admin).

Reply:
I don't know the answer to these questions unfortunately, I co-maintain the Galaxy-part of BioBlend but never really used CloudMan.

Reply:
`ubuntu` is the one and only username that worked with CloudMan<2.0 regardless of the cloud so we can just keep that here as the default. The new CM is using `admin` but it's proxied via Keycloak and I'm not really sure if this code will even work with it so if it's working for you @edwardsnj, it's fine to keep it like so until/if a larger update comes along.